src/go/rpk/pkg/cli/cmd/cluster/license/set.go

```diff
@@ -0,0 +1,49 @@
+package license
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newSetCommand(fs afero.Fs) *cobra.Command {
```

Comment:
question: Was the idea that `--path` is optional and you can also pass it directly on the command line?

Reply:
I think we agree on using `--path` and now I realize that passing it directly on the command line will make the license leak into shell history so it's better to use a file, wdyt?

Reply:
Yes IMO it being optional means you can have the benefits of both.

Good point about `--password`

The license key itself is never searched for in the env. In this PR when you see references to that, that is a unit test looking for a test license in the CI environment.

Reply:
Ok, so --path should be the default and then allow another optional flag to pass in the license, wdyt of:

`rpk cluster license set --license <base64 encoded license>`

or `--raw-license`

Reply:
Why not `rpk cluster license set <base64 encoded>`?

Reply:
Future proof and easier to deprecate: if we add another command (not likely) after `license set` it will be confusing to make the arg validation. Also, I was thinking that if the preferred way will be `--path` then it will be confusing to explain that the argument is optional.